department-of-veterans-affairs/abd-vro
# Add svc-bgs-api integration tests as a Gradle task

## What was the problem?

Integration tests were implemented within our GitHub actions workflow which prevented them from being run locally and also created undue complexity.

Associated tickets or Slack threads:
- #2172 

## How does this fix it?[^1]

Converts svc-bgs-api integration tests to 

## How to test this PR
- Spin up the platform docker containers along with the BGS container and run `./gradlew :svc-bgs-api:integrationTest`
- Check for successful run of the `svc-bgs-api` integration test GH workflow on this PR


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
